test/form.js

```diff
@@ -162,7 +180,7 @@ describe('Form', function () {
       this.form.submitHandler = 'fakeHandler';
       this.form.detach();
 
-      expect(this.form.submitHandler).to.equal(undefined);
+      expect(this.form.submitHandler).to.not.exist;
```

Comment:
This is slightly different behavior. Could we keep it as `.to.equal(undefined)`?